client/templates/profile/self/self.html

```diff
@@ -66,6 +66,20 @@
         {{/each}}
       </ul>
     </div>
+    <div class="row">
+      <h4>Your initiative:</h4>
+      <ul>
+        {{#each userInitiatives}}
+        <li>
+          <a href="{{pathFor 'initiative' slug}}">{{title}}</a>
+        </li>
+        {{else}}
+        <li>
+          <p>You have no initiatives... <a href="{{pathFor 'initiatives'}}">Create your first initiative!</a></p>
```

Comment:
This takes the user to the landing page. It should probably take them to the create page intead.